sql/lesson5/tutorial.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: page
+title: Introduction to SQL
+---
+
+## About Lesson 5
+
+The previous lessons showed you how to manipulate data from a single table. _Lesson 5_ will show you how to combine data from multiple tables and will bring out the _relational_ part of relational databases.
+
+Tables in a relational database relate to each other as they contain keys which allow information from multiple tables to be combined easily. The real power of relational databases, and SQL, comes from working with multiple tables at once.
```

Comment:
May I suggest a slight alteration, to use existing terminology and explain the concept a little clearer? Something along the lines of:
`Tables in a relational database are related by having common values in the fields of the tables. These fields are known as keys. These allow information from multiple tables to be combined...`

Reply:
Done